guidelines/REUSE.md

```diff
@@ -275,7 +203,7 @@ To include these parameters, you would need to add them individually as such:
 
 Again, Operations can be shared across files.  Although the reusability of operations will be less than with Parameters and Definitions. For this example, we will share a common `health` resource so that all APIs can reference it:
```

Comment:
> For this example

The example is missing...might be best to eliminate this intro, or move it to `JSON_References.md`